tests/test_trimming.py

```diff
@@ -36,26 +36,64 @@ def test_soundclasses():
 @pytest.mark.parametrize(
     'alms,kw,result',
     [
-        (["abc", "a-c", "--c"], {}, 'ac'),
-        (["abc", "a-c", "--c"], dict(skeletons=['VCC']), 'abc'),
-        (["a+bco", "-+cco", "-+cco"], {}, 'bco'),
-        (["a+b", "-+c", "-+c"], dict(exclude=""), 'a+b'),
+        (["abc", "a-c", "--c"], {}, list('ac')),
+        (["abc", "a-c", "--c"], dict(skeletons=['VCC']), list('abc')),
+        (["a+bco", "-+cco", "-+cco"], {}, list('bco')),
+        (["a+b", "-+c", "-+c"], dict(exclude=""), list('a+b')),
+        ([
+             #"- - n u - - 'b/b a".split(),
+             '- - - - d ù/u - -'.split(),
+             '- - - - d ú/u - -'.split(),
+             '- - - - d ù/u - -'.split(),
+             "ɾ u 'w/w a s i ɾ a".split(),
+             '- - - - s u - e'.split(),
+             "- - n u - - 'b/b a".split(),
+             '- - - - d u l -'.split(),
+             '- - n u k - w ɔ'.split(),
+         ], {}, ['d', 'ù/u']),
+        ([
+             "- - n u - - 'b/b a".split(),
+             '- - - - d ù/u - -'.split(),
+             '- - - - d ú/u - -'.split(),
+             '- - - - d ù/u - -'.split(),
+             "ɾ u 'w/w a s i ɾ a".split(),
+             '- - - - s u - e'.split(),
+             #"- - n u - - 'b/b a".split(),
+             '- - - - d u l -'.split(),
+             '- - n u k - w ɔ'.split(),
+         ], {}, ['-', '-']),
+        # Non-overlapping alignments:
+        (['- - a b'.split(), 'a b - -'.split(), 'a b - -'.split()], {}, ['-', '-']),
+        #
+        (['- a b'.split(), 'b a -'.split(), 'b a -'.split()], {}, ['-', 'a']),
+        (['- a b'.split(), 'b - -'.split(), 'b - -'.split()], {}, ['-', 'a']),
```

Comment:
@LinguList Here's the relevant test: The last site - having the same gap ratio as the middle site - is trimmed.